blog/_posts/2021-11-12-ahed-project.md

```diff
@@ -0,0 +1,392 @@
+
+The project described in this blog is a collaboration with NASA’s Astrobiology Habitable Environments Database (AHED) to assess the bibliographic holdings of AHED, identify those that have already been included in ADS’s holdings, flag those that are not yet included, and finally curate records and ingest those missing into ADS. This is one example of ADS’s goal to form collaborations with partners and expand access to scientific data and literature. The results of this project will be a first indication of how well the ADS covers the literature relevant for this type of research.
+
+As a recent addition to the ADS Team supporting curation efforts and assisting in collection management, this appealed to my interests as I have recently begun honing my Python skills, and learning new tools such as Jupyter Notebook and OpenRefine. Jupyter Notebook is especially useful for new Python users because it helps break up scripts into more manageable blocks (cells) and I can include my notes, findings, and documentation along the way.
+ 
+In this documentation I intend to outline the goals I established, the steps I took to accomplish them, and lessons learned. To accomplish this project, I used a combination of my own knowledge and expertise, I read API Documentation, searched online for solutions as needed, and I also collaborated with team members to debug and learn the ins and outs of the ADS API.
+
+### Project Outline and Goals
+
+The source data used in this project was an excel spreadsheet provided of AHED’s bibliographic holdings (with metadata for Authors, Org code, Title, Journal information, and DOI if available), which was split up into three sheets by Branch (Astrophysics Branch, the Planetary Systems Branch, and the Exobiology Branch). 
+
+<b>Sample image of original excel file, showing items in three tabs for the AHED Branches</b>
+
+<img width="1004" alt="Screen Shot 2021-11-04 at 11 44 23 AM" src="https://user-images.githubusercontent.com/31739067/140366721-e2721366-94d7-4651-a06d-17e7a10483e4.png"> 
+
+The main overall goal was to match each publication with an ADS bibcode if it exists. In order to accomplish this, my team recommended I use the ADS API to match based on DOIs (for those provided), then reference strings, and then fill in the rest by Title. I split up my overall goal into four major tasks, and for each of these phases I created a new Jupyter Notebook and outlined the steps.
+1. [(Notebook 1): Match AHED to ADS Items by DOI](#match-by-doi)
+2. [(Notebook 2): Match AHED to ADS Items by Reference Strings](#match-by-ref)
+3. [(Notebook 3): Match AHED to ADS Items by Title](#match-by-title)
+4. [(Notebook 4): Curate missing items and create ADS Libraries](#ads-libs)
+
+## <a name="match-by-doi">Task 1 (Notebook 1): Match AHED to ADS Items by DOI</a>
```

Comment:
After reading through the whole thing, I think it'd be more clear if you removed "(Notebook X)" from all 4 of your section headers, so they read more like "Task 1: Match AHED to ADS Items by DOI". And change the 4-point list just above to read "Task X" instead of "Notebook X" for each item. Most people aren't going to look at the notebooks, so that'll make it a bit more understandable.

Also, I might have missed it, but do you link to the notebooks anywhere? Even if you do, I'd make it more prominent. Maybe just below the section header, you could write something short in italics like this:

*Code snippets in this section are taken from AHED Project Notebook 1.*

And link "AHED Project Notebook 1" to the appropriate notebook. I'd repeat this in every section